protobuf/protoc_echo_plugin/Main.cpp

```diff
@@ -8,7 +8,8 @@
 int main(int argc, char* argv[])
 {
 #ifdef _MSC_VER
-    //_CrtDbgBreak();
+    //_CrtSetReportMode(_CRT_WARN, _CRTDBG_MODE_WNDW);
+    //_CrtDbgReport(_CRT_WARN, nullptr, 0, nullptr, "Attach Visual Studio with Debug -> Attach to Process before pressing Retry");
```

Comment:
Do you still would like to keep these commented ? Looks like a nice warning when using VS

Reply:
Yes, these need to be commented, otherwise when built in debug they don't run but display a crash window instead. You uncomment these when trying to debug the proto plugin